### ISSUE TITLE
We need to find another source for PT-BR mangas

### ISSUE THREAD

**Issue by weigler** [aplanas/kmanga#16]
I really enjoy the Kmanga app/site and don't want to stop the use. I can read in English of course but it's easier in my mother language. How I can help you?

**Comment by aplanas** [comment]
Yeah, another user give me this source as a good one: https://mangalivre.com

Any recommendation?

**Comment by MaximianoEduardo** [comment]
I know another source: https://mangahost-br.cc/

**Comment by weigler** [comment]
I tried to contact some sources... if I remember well, was the same that one of you post... the problem will be the authorization from the source because not every have good servers, right?

**Comment by aplanas** [comment]
Another alternative (suggested by maidenmetal via email)

http://cdmnet.com.br

**Comment by aplanas** [comment]
Another one, recommended by an user:

https://yesmangasbr.com/

**Comment by weigler** [comment]
Someone accept?
